app/styles/brands/psyarxiv.scss

```diff
@@ -25,5 +25,5 @@
 
 .branded-footer a,
 #view-page .osf-box-lt .fa {
-    color: #ff383f;
+    color: #CF1D35;
```

Comment:
The JIRA ticket describes the situation as follows:
> The buttons and logo colors were changed, but the footer did not get updated to their new color.

This sounds like a great use case for [SASS variables](http://sass-lang.com/guide#topic-2), which would let us define the color in just one place that could be more easily maintained in the future.